src/toggle-switch/toggle-switch.scss

```
@@ -0,0 +1,235 @@
+.ToggleSwitch {
+  align-items: center;
+  display: inline-flex;
+
+  &:hover {
+    .ToggleSwitch-knob {
+      background-color: var(--color-btn-hover-bg);
+    }
+  }
+
+  &:active {
+    .ToggleSwitch-knob {
+      background-color: var(--color-btn-active-bg);
+    }
+  }
+}
+
+.ToggleSwitch-track {
+  position: relative;
+  display: block;
+  width: 64px;
+  height: 32px;
+  padding: 0;
```

We decided to clip the shadow to be "inside" the toggle switch. [The PR](https://github.com/primer/react/pull/2057) just hasn't merged yet.

```suggestion
    padding: 0;
    overflow: hidden;
```